keyboards/2d4d/config.h

```diff
@@ -0,0 +1,70 @@
+// Copyright Mehmet DEĞİRMENCİOĞLU (@mechMehmet) <https://www.youtube.com/@mechmehmet>
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#include "config_common.h"
+
+#define MATRIX_ROWS 8  // Rows are doubled-up
+#define MATRIX_COLS 6
+
+#define DEBOUNCE 5
+
+/* COL2ROW, ROW2COL*/
+#define DIODE_DIRECTION COL2ROW
+
+/* Mechanical locking support. Use KC_LCAP, KC_LNUM or KC_LSCR instead in keymap */
+#define LOCKING_SUPPORT_ENABLE
+/* Locking resynchronize hack */
+#define LOCKING_RESYNC_ENABLE
+
+//sırayla yaz
+#define MATRIX_ROW_PINS \
+    { B3, B4, B5, B6, A2, A3, A4, A5 }
+#define MATRIX_COL_PINS \
+    { B12, B13, B14, B15, A8, A15 }
+
+//A VE B kanalı çok önemli değil, eğer yanlış bağlarsan bunları yer değiştir.
+#ifdef ENCODER_ENABLE
+#define ENCODERS_PAD_A \
+    { A1 }
+#define ENCODERS_PAD_B \
+    { A0 }
+#define ENCODER_RESOLUTION 4
+#endif
```

Comment:
All this is required to be configured in `info.json`. Please see [PR checklist](https://docs.qmk.fm/#/pr_checklist?id=keyboard-prs) for more info. (This file should be removed after doing so)